x-pack/test_serverless/api_integration/test_suites/observability/index.feature_flags.ts

```diff
@@ -10,5 +10,11 @@ import { FtrProviderContext } from '../../ftr_provider_context';
 export default function ({ loadTestFile }: FtrProviderContext) {
   describe('Serverless observability API - feature flags', function () {
     loadTestFile(require.resolve('./custom_threshold_rule'));
+    describe('Infra UI', function () {
+      loadTestFile(require.resolve('./infra/metadata'));
```

Comment:
Is there a reason why you removed the index file and now load the test files one by one here?

Reply:
Because I want them to run under the feature flag config and I moved them in the [index.feature_flags.ts](https://github.com/elastic/kibana/pull/168818/files/b47ad386ac0eac3dfd8f15fb0b513f34084653b2#diff-fdf1fb8761bd5fbba5b3bbb627bfa55d579b1ec9457dbf98ac69fead22267329)  (if I leave the index file there they will run together with the not feature flag tests) Why do I need a separate index? Am I missing something here 🤔

Reply:
Moved the index back, thanks 👍